docs/pages/management/guides/teleport-operator.mdx

```diff
@@ -265,6 +271,19 @@ has been locked due to a certificate generation mismatch, possibly indicating a
 In order to resolve this issue, remove any Teleport Kubernetes Operator sidecars connecting to the shared backend that are not in the same
 Kubernetes cluster or namespace.  Then redeploy the Auth Server pods.  When the operator starts, it deletes the cached information and recreates all required resources.
 
+### Teleport resource name is invalid with the Operator
+
+Kubernetes enforces all custom resource names to follow RFC 1123.  If attempting to create a resource using the operator with a name that is invalid, you will
```

Comment:
I think it would be good to add a quick note after the first sentence to indicate the scope of RFC 1123 (e.g., ", which includes specifications for hostnames") to save readers a minute of Googling if they don't know what this means already.